theta/smartsim_theta_installer.sh

```diff
@@ -23,5 +23,5 @@ make lib && pip install .
 cd ..
 cd smartsim
 conda install swig cmake git-lfs -y
-pip install .[ml]
-smart --device cpu --onnx
\ No newline at end of file
+pip install -e .[dev] keras==2.3 tensorflow==2.4.2 onnx==1.9.0
```

Comment:
Can we update this to just run `pip install .[ml]`?

Reply:
Adressed.